cmd/init.go

```diff
@@ -403,7 +403,7 @@ func init() {
 
 	// System Configuration Flags based on previous system_config.yml and networks_derived.yml
 	initCmd.Flags().String("system-name", "sn-2024", "Name of the System")
-	initCmd.Flags().String("csm-version", "1.3", "Version of CSM being installed")
+	initCmd.Flags().String("csm-version", "1.4g", "Version of CSM being installed")
```

Comment:
what's the `g` for?

Reply:
LTE?

Reply:
Switched to a terminal and typed `git` but was too fast for my laggy Mac.